tests/testing_utils.py

```diff
@@ -432,3 +443,13 @@ def optimize_with_hotstart(self, tol, optOptions=None, x0=None):
         # we should have zero actual function/gradient evaluations
         self.assertEqual(self.nf, 0)
         self.assertEqual(self.ng, 0)
+
+    def tearDown(self):
+        # remove history file
```

Comment:
What is missing here? It looks like it is covering all the generated files

Reply:
Didn't get rid of all the files for me when I tried this. Don't have time to fix this though.